src/main/java/hotel/models/Booking.java

```diff
@@ -0,0 +1,107 @@
+package hotel.models;
+
+import customexceptions.InvalidHotelActionException;
+
+import java.time.LocalDate;
+
+/**
+ * This class serves as property
+ * of each Room object - it allows
+ * reservation dates to be updated and
+ * it can be added to set
+ *
+ * @author Ivan Velkushanov
+ */
+public class Booking {
+	private static int lastNumberUsed;
+	private final int bookingNumber;
+	private LocalDate fromDate;
+	private LocalDate toDate;
+	private String guestName;
+	private long guestId;
+
+	/**
+	 * Constructor of the Booking class
+	 *
+	 * @param fromDate
+	 * @param toDate
+	 * @param guestId
+	 */
+	public Booking(LocalDate fromDate, LocalDate toDate, long guestId) {
+		changeReservationDates(fromDate, toDate);
+		guestName = "unknown";
```

Comment:
Why unknown is used? it is maybe better to have another constructor with all properties like `public Booking(LocalDate fromDate, LocalDate toDate, String guestName, long guestId) `which will be invoked in this one with the parameter `unknown`. Other than that is not good practice to keep string like this you can create a constant as: `private static final String UNKNOWN_NAME = "unknown";`